mono/utils/mono-stdlib.c

```diff
@@ -35,7 +35,12 @@ mono_mkstemp (char *templ)
 
 	len = strlen (templ);
 	do {
+#if HOST_WIN32
+		t = _mktemp (templ);
```

Comment:
Add `g_mktemp`?

Reply:
I'll add it.

Reply:
I've fixed this at 71301d55bfd1b714f6535eae72ba11b5cbca07c3.